Extension/src/abTesting.ts

```diff
@@ -61,6 +67,12 @@ export class ABTestSettings {
         return util.isNumber(this.settings.gotoDefIntelliSense) ? this.settings.gotoDefIntelliSense >= this.bucket.Value : true;
     }
 
+    public minimumVscodeSupportedVersion(extensionVersion: string): string {
+        return (this.settings.supportExtensionVscodeVersion && this.settings.supportExtensionVscodeVersion[extensionVersion]) ?
```

Comment:
With only entries for the dependency changes, this will need to be reworked a bit.
Iterate through the keys in the map and for each one, if it is less than the `extensionVersion`, make sure that the current vscode version is greater than the value for that key.  You'll want to create PackageVersion objects for these comparisons since we already have a comparison function in that class.